src/main/js/field/input/TableSelectInput.js

```diff
@@ -255,109 +254,140 @@ class TableSelectInput extends Component {
 
     createTable() {
         const columnsProp = this.props.columns;
-        const defaultSortName = columnsProp.find(column => column.sortBy).header;
+        const defaultSortName = columnsProp.find((column) => column.sortBy).header;
 
         const tableOptions = {
             noDataText: 'No data found',
             clearSearch: true,
-            defaultSortName: defaultSortName,
+            defaultSortName,
             defaultSortOrder: 'asc'
         };
 
         const projectsSelectRowProp = this.createRowSelectionProps();
 
         const assignDataFormat = (cell, row) => {
-            const cellContent = (row.missing && cell && cell !== '') ?
-                <span className="missingData">
-                    <FontAwesomeIcon icon="exclamation-triangle" className="alert-icon" size="lg" />{cell}
-                </span>
+            const cellContent = (row.missing && cell && cell !== '')
+                ? (
+                    <span className="missingData">
+                        <FontAwesomeIcon icon="exclamation-triangle" className="alert-icon" size="lg" />
+                        {cell}
+                    </span>
+                )
                 : cell;
 
             if (cell) {
-                return <div title={cell.toString()}> {cellContent} </div>;
+                return (
+                    <div title={cell.toString()}>
+                        {' '}
+                        {cellContent}
+                        {' '}
```

Comment:
These empty spaces are strange, do you know why they were inserted?

Reply:
Looks like they were there in the original, but ESLint wants it to be explicit?

Reply:
The dive has space before an after the cellContent so the linter to preserve those whitespace characters added the `{' '}` code to preserve the spacing